src/main.rs

```diff
@@ -14,8 +15,17 @@ struct Args {
   #[arg(short, long)]
   pub verbose: bool,
 
-  #[arg(short = 'w', help = "Skip compilation warnings")]
-  pub skip_warnings: bool,
+  #[arg(
+    short = 'W',
+    value_delimiter = ' ',
+    action = clap::ArgAction::Append,
+    long_help = r#"Skip compilation warnings
```

Comment:
`-W` skipping warnings is unintuitive. Better to have -Wno-<warning> for skipping.